package.json

```diff
@@ -8,7 +8,9 @@
   "dependencies": {
     "css-layout": "^1.1.0",
     "d3": "^3.5.4",
+    "d3-path": "^0.1.5",
```

Comment:
I think it's correct as a devdep.

We use an interface that both d3-path and Context2D implement. Therefore we
don't have a direct dependency on either. However an easy way to write the
tests is to use d3 -path to serialise to an svg path string for assertions.

Let me know if that's still not clear.

On Tue, 1 Mar 2016 17:44 Colin Eberhardt, <notifications@github.com> wrote:

> In package.json
> <https://github.com/ScottLogic/d3fc/pull/919#discussion_r54604132>:
>
> > @@ -8,7 +8,9 @@
> >    "dependencies": {
> >      "css-layout": "^1.1.0",
> >      "d3": "^3.5.4",
> > +    "d3-path": "^0.1.5",
>
> Having it as a dev dependency sounds wrong to me. It is a requirement for
> usage of d3fc-path, not just development.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ScottLogic/d3fc/pull/919/files#r54604132>.
>